quickstarts/flutter-quickstart/config.yml

```
@@ -0,0 +1,62 @@
+slug: flutter-quickstart
+
+# Used as the display name for the quickstart
+title: Flutter quickstart
+
+description: |
+  ## Why monitor Flutter
+
+  Flutter is one of the fastest growing cross platform frameworks for building iOS, Android, Windows, Mac, Web, and Linux applications.
+
+  This dashboard collects metrics from the Native iOS and Android integrations and composes a single dashboard to compare both applications performances.
+
+  ### Flutter quickstart highlights
+
+  Dashboard:
+  - Monitor session duration between all platforms.
+  - Track crashes, function exceptions, and memory usage.
+  - Easily add your own metrics to the dashboard using the `Flutter` custom event handle.
+
+  Alerts:
+  - Memory usage.
+  - Function exceptions.
+  - Network Latency.
+
+
+summary: |
+  Quickstart for Flutter utilizing native iOS and Android agents
+
+# Possible values: New Relic | Verified | Community
+# Please consult with pull request reviewers if you think your quickstart should have a support level other than "Community"
+level: Community
+
+authors:
+  - Jake Landers
+  - Hitesh Ahuja
+  - Kartik Pattaswamy
+  - Allen Wang
+  - Claire Howard
+  - Zyria Lott
+
+keywords:
+  - flutter
+  - mobile
+  - crossplatform
+  - ios
+  - android
+
+# An icon or logo for the quickstart
+icon: flutter.svg
+
+# References to dashboards by file path, the first entry references a dashboard located at `dashboards/example-dashboard` 
+# For more information on dashboard definitions, see the section below.
+dashboards:
+  - flutter
+
+# References to alert policies by file path, the first entry references an alert policy located at `alerts-policies/example-alert-policy` 
+# For more information on alert policy definitions, see the section below.
+alertPolicies:
+  - flutter
```

This will also need to match the path
```suggestion
  - flutter-alert-policy
```